05. Assignment/README.md

```diff
@@ -0,0 +1,52 @@
+# Assignment 1 - GitHub Collaboration Fundamentals
+
+## Description
+
+This assignment introduces new team members to GitHub collaboration workflow. You will learn how to invite collaborators to a repository, create pull requests, and add feedback through PR comments and reviews.
+
+## Use Cases
+- Onboarding new developers
+- Practicing basic GitHub contribution workflow
+- Understanding the workflow of submitting PRs and leaving review comments
+  
+## Assignment Details
+### 1. Invite a user to GitHub repository
+
+**Steps:**
+- Open `Settings` → `Collaborators & teams` in your repository
+- Click `Add people`
+- Invite teammates with their GitHub username or email
+- Teammate will receive an invitation email to join the repository
+![assigment1_add_collaborator.jpg](images/a1_add_collaborator.png)
+
+### 2. Create a Pull Request
+
+**Steps:**
+- Clone repository to local machine
+- Create a feature branch
+- Add any updates (e.g: edit README or add a file)
+- Commit and push the changes
+![assignment1_creating_pr.png](images/a1_creating_pr_1.png)
+
+- Fill out PR title and description
+- Review your branch, target branch, and code changes before opening PR
+![assignment1_creating_pr_2.png](images/a1_creating_pr_2.png)
+![assignment_1_creating_pr_3.png](images/a1_creating_pr_3.png)
+
+### 3. Comment on teammate’s PR
+
+**Steps:**
+- Navigate to the `Pull requests` tab inside the repository
+- Select the PR and go to `Files changed` tab and review
+- Add comment(s) if needed
+![assignment1_review_pr_1.png](images/a1_review_pr_1.png)
+
+**Note** 
+- `Add single comment` → add a comment immediately and notify the author
+- `Start a review` → allow you to add multiple comments and submit them together with action (Comment/Approve/Request changes)
+![assignment1_review_pr_2.png](images/a1_review_pr_2.png)
+
+## Expected Output
+- Screenshot: Invite another user to the repository
```

Comment:
the screenshots are mostly for reference so they cannot be consider as output
in this case, the expected output would be:
- successful collaboration invitation sent to teammate
- successful acceptance of teammate collaboration on github repo
- Pull Request created and asked for teammate's review
- Comments created on Pull Request